### PR TITLE
(feat mutant): Add vcf delivery tags

### PR DESCRIPTION
**Pull request by sofstam**
### Added

- vcf files to be delivered to KS

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b vcf_deliver -a
    ```

### How to test

- [x] `cg deliver analysis --dry-run --delivery-type mutant -c maturejay`

Screenshot 2024-02-14 at 12 43 36

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by karlnyr — APPROVED** [review]
Nice ✨

File: cg/constants/delivery.py [hunk 1]
```diff
@@ -145,6 +145,7 @@
 
 SARSCOV2_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
```
Comment (henrikstranneheim) [inline cg/constants/delivery.py:146]: Should this not be MUTANT?

**Review by henrikstranneheim — APPROVED** [review]
👍

File: cg/constants/delivery.py (hunk 1 above)
Comment (sofstam) [reply cg/constants/delivery.py:146]: Just added the tags. Shall I rename to `MUTANT_ANALYSIS_SAMPLE_TAGS` instead?

Seems that this constant is only being used in this repo. Is there another repo that needs to be updated though?

Comment (henrikstranneheim) [reply cg/constants/delivery.py:146]: That would be great. Only used in this repo. 👍

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by sofstam** [comment]
Deployment on stage:
Screenshot 2024-02-19 at 10 03 19

**Comment by sofstam** [comment]
Deployment on production:

Screenshot 2024-02-19 at 10 06 09

**Pull request merged**